implementation.md

```diff
@@ -1964,3 +1964,35 @@ While SSPs and DSPs may find use for Extended Content IDs, it is also perfectly
   }
 }
 ```
+
+## 7.15 - Additional Telemetry for Live Events <a name="lea"></a>
+
+Additional telemetry can be provided by including `recording` and `airdate` attributes. There are some detailed examples below.
+
+- The first airing of a program or event that has played previously on a different channel should use the date the program originally ran (e.g. a show new to channel B should include the original airdate as it ran on channel A).  
+- Where the original airdate is very far in the past (e.g. a classic movie streaming live), the value in the `airdate` attribute should be left blank.
+
+---
+
+## Example Scenarios
+
+| Scenario | `recording` | `airdate` |
```

Comment:
See recommendations on special `airdate` values -1 and -2 and consider updating this table.